.env.example

```diff
@@ -2,7 +2,15 @@
 DEBUG=1
 SECRET_KEY=foo
 DJANGO_ALLOWED_HOSTS=localhost 127.0.0.1 [::1]
+DJANGO_LOG_LEVEL=INFO
 
 # Catalogue Variables
 CATALOGUE_TOKEN=mycatalogtoken
 CATALOGUE_URL=https://datahub-catalogue-dev.apps.live.cloud-platform.service.justice.gov.uk/api/gms
+
+# Azure Variables (Examples)
+AZURE_AUTH_ENABLED=true # any string value other than true to disable Azure authentication
+AZURE_CLIENT_ID=< obtain from onepassword >
```

Comment:
I think in one of the other projects we had a nice script that generated the .env based by pulling all the secrets from secrets manager. It's not essential right now, but just wondering if we could do the same here so when someone joins the team they just have run one command for this?

Reply:
Yes I am all in favour of a script that pulls secrets from secret manager and builds the .env dynamically. Perhaps introduce a Makefile at the same time. Ill create a ticket for it.